.github/workflows/deploy.yml

```
@@ -0,0 +1,72 @@
+name: Deploy
+
+on: 
+  push:
+    branches:
+      - main
+    
+jobs:
+  deployment:
+
+    runs-on: ubuntu-latest
+    environment: Production
+    steps:
+    - name: Get current time for Folder
+      uses: josStorer/get-current-time@v2.0.1
+      id: current-time
+      with:
+        format: YYYYMMDDHHMMSS
+    - name: Get current time for Cleanup
+      uses: josStorer/get-current-time@v2.0.1
+      id: current-time-cleanup
+      with:
+        format: YYYYMM
+    - uses: actions/checkout@v1
```

v1 has security issues and hasn't been updated in years

```suggestion
    - uses: actions/checkout@v3
```